### PR TITLE
Document Ctrl + 1,2,3...9 shortcut

### LINKED ISSUE #989: Document Ctrl + 1,2,3...9 shortcut
This shortcut allows the user to jump from 1st to 9th issue using Control + 1,2,3..9 keys.

### PR DESCRIPTION
**Pull request by codemanmeet**
Fixes #989

### CODE REVIEW

File: docs/keyboardShortcuts.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ These key sequences (one key followed by the other) are used to load (in the bro
 | Filter box|<kbd>Ctrl</kbd> + <kbd>F</kbd>|Focus moves to the filter box of the selected panel|
 | Issue list|<kbd>Ctrl</kbd> + <kbd>Enter</kbd>|Focus moves to issue list of the selected panel|
 | Out of filter box (while editing filter)|<kbd>ESC</kbd> then <kbd>ESC</kbd>|First <kbd>ESC</kbd> restores previous filter and selects the filter text. The second <kbd>ESC</kbd> moves focus out of filter box|
+|n<sup>th</sup> issue (from *1* to *9*)| <kbd>Ctrl</kbd> + <kbd>n</kbd>| If the focus is already on the issue list, jumps to nth issue, where n is the issue number ranging from 1 (at the top of the list) to 9|
```
Comment (dariusf) [inline docs/keyboardShortcuts.md]: - Second column is misleading. The `n` shouldn't be in `<kbd>`. There should be a better way to communicate that we mean literal <kbd>1</kbd> to <kbd>9</kbd> keys.
- Missing superscript in the third column

File: docs/keyboardShortcuts.md [hunk 2]
```diff
@@ -73,4 +74,4 @@ These shortcuts can be used when the focus is on an issue card in a panel.
 | ------------- |:-------------:| -----------|
 | Scroll to top|<kbd>I</kbd>|Scrolls to the top of the page loaded in browser view|
 | Scroll to bottom|<kbd>N</kbd>|Scrolls to the bottom of the page loaded in browser view|
-| Scroll up/down|<kbd>J</kbd> / <kbd>K</kbd>|Scroll up/down the page loaded in browser view|
\ No newline at end of file
+| Scroll up/down|<kbd>J</kbd> / <kbd>K</kbd>|Scroll up/down the page loaded in browser view|
```
Comment (dariusf) [inline docs/keyboardShortcuts.md]: What changed here?

Comment (codemanmeet) [inline docs/keyboardShortcuts.md]: Yup even I'm surprised why Github is displaying this as a change. There seems to be no change. It's exactly the same thing.

**Comment by dariusf** [comment]
Okay, please squash. We don't have to wait for Travis for this one.

**Comment by codemanmeet** [comment]
Now I get it. I removed a trailing space at the end!

**Pull request merged**